config/300-servicebinding.yaml

```diff
@@ -212,3 +212,192 @@ spec:
     storage: true
     subresources:
       status: {}
+  - additionalPrinterColumns:
```

Comment:
from https://github.com/servicebinding/spec/blob/v1.0.0/servicebinding.io_servicebindings.yaml#L19-L207

Only change is that we're keeping v1alpha3 as the stored version.